core/src/main/java/org/infinispan/remoting/transport/jgroups/CommandAwareRpcDispatcher.java

```diff
@@ -55,6 +56,7 @@
    private static final Log log = LogFactory.getLog(CommandAwareRpcDispatcher.class);
    private static final boolean trace = log.isTraceEnabled();
    private static final boolean FORCE_MCAST = Boolean.getBoolean("infinispan.unsafe.force_multicast");
+   private static final long STAGGER_DELAY_NANOS = TimeUnit.MILLISECONDS.toNanos(Long.getLong("infinispan.stagger.delay", 5));
```

Comment:
While the shorthand Type.getType() methods are nice to use, we really need to use SecurityActions to retrieve system properties.
Also, please create a documentation section to hold all the system environment properties that affect Infinispan.
I have found:
infinispan.arrays.debug
infinispan.unsafe.force_multicast
infinispan.compat
infinispan.debugDependencies